media_driver/linux/ult/libdrm_mock/xf86drm_mock.c

```diff
@@ -357,6 +366,12 @@ mosdrmIoctl(int fd, unsigned long request, void *arg)
             ret = -1;
         }
         break;
+        case DRM_IOCTL_I915_GEM_VM_CREATE:
+        case DRM_IOCTL_I915_GEM_VM_DESTROY:
+        {
+            ret = 0;
```

Comment:
For full compatibility with i915 you might want to set vm_id to non-zero. (i915 will never provide zero vm_id.) I know it is not documented, but it may make sense regardless so there is some differentiation between unitialized struct and returned data.

Reply:
fixed.